packages/legend-extension-dsl-service/src/stores/studio/UpdateServiceQuerySetupStore.ts

```diff
@@ -112,12 +124,71 @@ export class UpdateServiceQuerySetupStore {
     this.applicationStore.navigator.goTo(
       generateServiceQueryUpdaterSetupRoute(undefined, undefined, undefined),
     );
+    this.resetProjectConfiguration();
   }
 
   resetCurrentGroupWorkspace(): void {
     this.currentGroupWorkspace = undefined;
     this.currentWorkspaceService = undefined;
   }
+  setIsCurrentProjectNotConfigured(val: boolean | undefined): void {
+    this.isCurrentProjectNotConfigured = val;
+  }
+
+  setCurrentProjectConfigurationReviewUrl(val: string | undefined): void {
+    this.currentProjectConfigurationReviewUrl = val;
+  }
+
+  resetProjectConfiguration(): void {
+    this.setCurrentProjectConfigurationReviewUrl(undefined);
+    this.setIsCurrentProjectNotConfigured(undefined);
+  }
+
+  *setCurrentProjectConfiguration(): GeneratorFn<void> {
+    if (this.currentProject) {
+      this.setIsCurrentProjectNotConfigured(
+        !((yield this.sdlcServerClient.isProjectConfigured(
+          this.currentProject.projectId,
+        )) as boolean),
+      );
+      if (this.isCurrentProjectNotConfigured) {
+        this.applicationStore.notifyIllegalState(
+          'Current project is not configured',
+        );
+      }
+      yield flowResult(this.fetchProjectConfigurationReviewUrl());
+    } else {
+      this.resetProjectConfiguration();
+    }
+  }
+
+  *fetchProjectConfigurationReviewUrl(): GeneratorFn<void> {
+    if (this.currentProject && this.isCurrentProjectNotConfigured) {
```

Comment:
just making sure here, you are attemping to fetch the corresponding `ProjectConfiguration` review set up by import project?
Do you filter in sdlc by the name of the source ?

Reply:
Yes in sdlc i filter review by source